src/_shared/scss/_advert.scss

```diff
@@ -19,6 +19,11 @@
     }
 }
 
+a.advert {
```

Comment:
Sorry, you can add the `blink` class to the `advert` and then `blink__anchor` to the `advert__title`